mlflow/store/artifact/azure_data_lake_artifact_repo.py

```diff
@@ -78,11 +81,11 @@ def __init__(self, artifact_uri, credential):
         self.credential = credential
         self.write_timeout = MLFLOW_ARTIFACT_UPLOAD_DOWNLOAD_TIMEOUT.get() or _DEFAULT_TIMEOUT
 
-        (filesystem, account_name, path) = _parse_abfss_uri(artifact_uri)
+        (filesystem, account_name, region_suffix, path) = _parse_abfss_uri(artifact_uri)
 
         # TODO: investigate setting the account URL based on whether the abfss URI is associated
         # with an Azure account in standard Azure, govcloud, mooncake, etc
-        account_url = f"https://{account_name}.dfs.core.windows.net"
+        account_url = f"https://{account_name}.dfs.core.${region_suffix}"
```

Comment:
the `$` is for Scala only I believe. This would give `...dfs.core.$windows.net` or ...`dfs.core.$chinacloudapi.cn`.
```suggestion
        account_url = f"https://{account_name}.dfs.core.{region_suffix}"
```

Reply:
Good catch, this needed to be fixed for manual QA to succeed